doc/tutorial/pairwise_alignment/index.md

```diff
@@ -243,6 +243,11 @@ a simple scoring scheme with mismatch `-2` and match `4`.
 
  -  TTACGTACGGACTAGCTACAACATTACGGACTAC
  -  GGACGACATGACGTACGACTTTACGTACGACTAGC
+
+\hint
+An alignment is called overlap alignment if it allows free end-gaps at each sequence end. In this configuration
+overlaps between two sequences can be computed which is a common use case during the overlap layout consensus assembly.
```

Comment:
"overlap layout consensus assembly"? Interesting chain of words xD

Reply:
Well, it is what it is, right? :)